### PR TITLE
Update config.

### PR DESCRIPTION
**Pull request by Namibillow**
Modified some of the cli arguments and code that used it.

### CODE REVIEW

File: utils/argparser.py [hunk 1]
```diff
@@ -1,13 +1,18 @@
+""" 
+    Collection of what arguments this project needs.
```
Comment (amillert) [inline utils/argparser.py]: `Collection of the arguments needed in the project.` - maybe sounds better?

File: corpus/preprocessing.py [hunk 2]
```diff
@@ -168,19 +184,25 @@ def _convert_group(self, row: str) -> int:
...
-        super(DataBuilder, self).__init__(args.path_corpus_out, args.load_data)
+    """Class to build collected texts to formatted data.
+
+    Args:
+        data (list): Contains raw text data. Pass None is only loading the data from saved ones.
+        path (str): Path to a directory to save/saved data.
+        load (bool, optional): Whether to load the saved data or not. Defaults to False.
+
+    """
+    def __init__(self, data: list, path : str, load=False):
```
Comment (amillert) [inline corpus/preprocessing.py]: def __init__(self, data: list, path: str, load: bool=False):

**Review by amillert — CHANGES_REQUESTED** [review]
Good job @Namibillow, thanks! Ready to merge after minor changes. I'll resolve conflicts.

File: utils/argparser.py (hunk 1 above)
Comment (Namibillow) [reply utils/argparser.py]: Sg.

**Comment by amillert** [comment]
@Namibillow, please provide the `corpus/persistance.py` file. I think You forgot it. I need it for merging.

File: main.py [hunk 3]
```diff
@@ -1,56 +1,52 @@
...
 
+from corpus.preprocessing import DataBuilder, FeaturesGenerator
+from handle_wiki.extraction import executeJob
+from prediction.clustering import classify, cluster
+from utils import queriesObject2Category
+from utils.argparser import args
+import traceback
 
-if __name__ == "__main__":
+def main(args):
```
Comment (amillert) [inline main.py]: doesn't work

Comment (Namibillow) [reply main.py]: Ok I guess put it back to the original.

**Pull request merged**